### PR TITLE
do not replace current timestamp with current year

### PR DESCRIPTION
**Pull request by ltclm**
Its my proposition to solve: https://github.com/geoadmin/mf-geoadmin3/issues/4192
Testlink with all time-enabled layers
https://mf-chsdi3.dev.bgdi.ch/ltclm/shorten/796dc058e0

<jenkins>[Test link](https://mf-geoadmin3.int.bgdi.ch/dev_ltclm_current_timeslider/index.html)</jenkins>

### CODE REVIEW

**Comment by oterral** [comment]
Problem with this PR is that you can't see the last version when the slider is activate.
[Test](https://mf-geoadmin3.dev.bgdi.ch/ltclm/src/?lang=de&topic=ech&bgLayer=voidLayer&layers=ch.swisstopo.lubis-luftbilder-dritte-firmen,ch.swisstopo.lubis-luftbilder-dritte-kantone,ch.swisstopo.lubis-luftbilder_farbe,ch.swisstopo.lubis-luftbilder_infrarot,ch.swisstopo.lubis-luftbilder_schwarzweiss,ch.astra.unfaelle-personenschaeden_getoetete,ch.astra.unfaelle-personenschaeden_fahrraeder,ch.astra.unfaelle-personenschaeden_alle,ch.astra.unfaelle-personenschaeden_motorraeder,ch.astra.unfaelle-personenschaeden_fussgaenger,ch.swisstopo.zeitreihen,ch.swisstopo.swissimage-product,ch.swisstopo.swissimage-product.metadata&E=2611306.73&N=1206013.10&zoom=10&api_url=%2F%2Fmf-chsdi3.dev.bgdi.ch%2Fltclm&layers_timestamp=,,99991231,99991231,99991231,99990101,99990101,99990101,99990101,99990101,18641231,2017,&layers_opacity=1,1,1,1,1,1,1,1,1,1,1,1,0.7&layers_visibility=true,true,false,false,false,false,false,false,false,false,false,true,false&time=2017)

**Comment by oterral** [comment]
I'm not against this PR if @davidoesch is agree

File: test/specs/time/TimeService.spec.js [hunk 1]
```diff
@@ -90,7 +90,6 @@ describe('ga_time_service', function() {
         expect(gaTime.getYearFromTimestamp('text')).to.be(undefined);
         expect(gaTime.getYearFromTimestamp(20131201)).to.be(undefined);
         expect(gaTime.getYearFromTimestamp('99991231')).to.be(undefined);
-        expect(gaTime.getYearFromTimestamp('current')).to.be(new Date().getFullYear());
```
Comment (oterral) [inline test/specs/time/TimeService.spec.js]: Leave the test, just replace `new Date().getFullYear()` by `undefined`

**Comment by ltclm** [comment]
thanks for the feedback @oterall
yes the ``current`` timestamp should not be available in the timeslider. for this use-case the timeslider is only covering the real timestamps ``YYYY``

**Comment by ltclm** [comment]
@davidoesch do you agree with this proposal. we want to deploy this next week with the swissimage hist publication.

**Comment by ltclm** [comment]
@oterral any idea why the new value is not translated when opening the application with 
``src/?...`` : https://mf-chsdi3.dev.bgdi.ch/ltclm/shorten/7954ea41fe

**Comment by gjn** [comment]
When I unsterdand correctly, for swissimage, when I have the timeslider active, I can't select 'current'?

**Comment by ltclm** [comment]
yes, current is not a real timestamp, its not available in the time slider.

**Comment by gjn** [comment]
AFAIK, we have 2 timeBehaviour:

all -> if this is set, then default is 'show all data timestamps'
latest -> if this is set, then default is 'show latest timestamps'.

In contect of swissimage, latest timestamp is 'current'.

On the other hand, we have a time parameter in the PL that shows current year of the timeslider. And it is (until now) always a year. So we can't cover current.

Can't we say that 'current' is always a pointer to latest year? Now 'current=2017' and next year 'current=2018'?

I think it's important to be able to share the timeslider with swissimage active and current being displayed - currently, we can't.

**Comment by ltclm** [comment]
> Can't we say that 'current' is always a pointer to latest year? Now 'current=2017' and next year 'current=2018'?

That's the reason for this pr, actually before this current has been replaced with the current year 2018 for the timeslider

now we have exactly the same behaviour for ``all`` and ``current``
* lubis layers have a pseudo timestamp ``99991231`` to display all data **this timestamp is not available in the timeslider**
* swissimage hist has ``current``to display the latest data ( automata) **this timestamp is not available in the timeslider**

**Comment by ltclm** [comment]
@oterral 
@davidoesch and @gjn agree with this proposal. 
can you please review the [last commit](https://github.com/geoadmin/mf-geoadmin3/pull/4204/commits/facff5ed669b736fa7a2201435cb1de906e70f0f)  and approve if you agree?

File: src/locales/empty.json [hunk 2]
```diff
@@ -273,11 +273,13 @@
   "aviation_service_link_href": "",
   "aviation_service_link_label": "",
   "time_all": "",
+  "time_current": "",
   "time_bt_disabled_tooltip": "",
   "time_select_year": "",
   "topic_aviation_tooltip": "",
   "aviation": "",
   "time_all": "",
+  "time_current": "",
```
Comment (oterral) [inline src/locales/empty.json]: `time_all` and `time_current` are defined twice. can you remove one please?.

**Comment by oterral** [comment]
Thx @ltclm

**Pull request merged**